### ISSUE TITLE
"\n" in description

### ISSUE THREAD

**Issue by fgallaire** [aerkalov/ebooklib#226]
As description is HTML, line breaks should not be converted in "\n" but ignored in the metadata Python string.

**Comment by aerkalov** [comment]
Good point. I noticed that description is handled as text after the tag.

In this case it will just get "text " as description.

```
<dc:description>text &lt; text2</dc:description>

```
